ui/components-react/src/test/propertygrid/MergingPropertyDataProvider.test.ts

```diff
@@ -0,0 +1,291 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import type { PrimitiveValue, PropertyRecord } from "@itwin/appui-abstract";
+import type { PropertyCategory } from "../../components-react";
+import { SimplePropertyDataProvider } from "../../components-react";
+import TestUtils from "../TestUtils";
+import { createMergedPropertyDataProvider } from "../../components-react/propertygrid/MergingPropertyDataProvider";
+
+describe("MergingPropertyDataProvider", () => {
+  const provider1Category: PropertyCategory = {
+    name: "provider1 Category name",
+    label: "provider1 category label",
+    expand: false,
+  };
+  const provider2Category: PropertyCategory = {
+    name: "provider2 Category name",
+    label: "provider2 category label",
+    expand: true,
+  };
+  const provider3Category: PropertyCategory = {
+    name: "provider3 Category name",
+    label: "provider3 category label",
+    expand: false,
+  };
+
+  const provider1Property: PropertyRecord =
+    TestUtils.createPrimitiveStringProperty(
+      "provider1 Property",
+      "provider1 Property raw"
+    );
+  const provider2Property: PropertyRecord =
+    TestUtils.createPrimitiveStringProperty(
+      "provider2 Property",
+      "provider2 Property raw"
+    );
+  const provider3Property: PropertyRecord =
+    TestUtils.createPrimitiveStringProperty(
+      "provider3 Property",
+      "provider3 Property raw"
+    );
+
+  let dataProvider1: SimplePropertyDataProvider;
+  let dataProvider2: SimplePropertyDataProvider;
+  let dataProvider3: SimplePropertyDataProvider;
```

Comment:
does the third provider provide any value to the tests?

Reply:
Removed third provider